src/kOS.Safe/Encapsulation/ISerializableValue.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace kOS.Safe.Encapsulation
+{
+    public interface ISerializableValue
```

Comment:
@tomekpiotrowski if you look at our binding classes or our functions. They are using our own Attributes.